### PR TITLE
Rename `helmfiles` to `releases`

### PR DESCRIPTION
**Pull request by goruha**
## What
* Rename `helmfiles` to `releases`

## Why
* Naming dir `helmfiles` was a mistake.

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ usage: |-
   ```
   FROM cloudposse/helmfiles:latest as helmfiles
 
-  COPY --from helmfiles /helmfiles/ /conf/helmfiles/
+  COPY --from helmfiles /helmfiles /conf/helmfiles/
```
Comment (aknysh) [inline README.yaml]:
```suggestion
  COPY --from=helmfiles /helmfiles /conf/helmfiles/
```

**Review by aknysh — COMMENTED** [review]
see comments

**Review by osterman — APPROVED** [review]

File: README.yaml (hunk 1 above)
Comment (osterman) [reply README.yaml]:
```suggestion
  COPY --from=helmfiles /helmfiles/ /conf/helmfiles/
```

**Review by osterman — CHANGES_REQUESTED** [review]
See comments

**Review by osterman — APPROVED** [review]

**Pull request merged**